packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -0,0 +1,338 @@
+import { attr, DOM, FASTElement, observable } from "@microsoft/fast-element";
+import { Option } from "../option";
+
+/**
+ * A Listbox Custom HTML Element.
+ * Implements the {@link https://www.w3.org/TR/wai-aria-1.1/#listbox | ARIA listbox }.
+ *
+ * @public
+ */
+export class Listbox extends FASTElement {
+    /**
+     * Disables the radio group and child radios.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ attribute: "disabled", mode: "boolean" })
+    public disabled: boolean;
+    private disabledChanged(): void {
+        if (!this.$fastController.isConnected) {
+            return;
+        }
+
+        if (this.disabled) {
```

Comment:
aria-disabled is bound in the template, I'm not sure you need to have this code.